src/Microsoft.AspNet.Mvc.Core/ActionResults/FileStreamResult.cs

```diff
@@ -39,7 +41,7 @@ public FileStreamResult([NotNull] Stream fileStream)
         /// </summary>
         /// <param name="fileStream">The stream with the file.</param>
         /// <param name="contentType">The Content-Type header of the response.</param>
-        public FileStreamResult([NotNull] Stream fileStream, string contentType)
+        public FileStreamResult([NotNull] Stream fileStream, MediaTypeHeaderValue contentType)
```

Comment:
In this case, are we responsible for re-encoding `fileStream` if it doesn't match the `Encoding` on `contentType`?